metadata_annotation.ipynb

```diff
@@ -0,0 +1,2466 @@
+{
```

Comment:
<ol><li>Please link to the publication with a markdown link.</li><li>Please introduce the dataset in 1 sentence. What kind of data does it have? How big? Which perturbations?</li><li>What is the <code>metadata object</code>? Be more precise. -&gt; "In pertpy, access to metadata databases is facilitated through<code>MetaData</code> objects that directly access the data bases through either the databases API or by caching their knowledge on-disk. We subsequently create a pertpy <code>Metadata</code> object to annotate cell lines. " Something along those lines.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>